src/components/interfaces.ts

```diff
@@ -14,6 +14,7 @@ export interface ComponentDefinition {
   regions: ComponentRegion[];
   functions: ComponentFunction[];
   events: EventHandler[];
+  types: Record<string, Record<string, never>>;
```

Comment:
Currently it is an empty object: `{ Variant: {} }`.

This is needed for future compatibility, if we need additional metadata inside this object